src/main/java/team/gif/robot/commands/autoaim/LimelightAutoAim.java

```diff
@@ -28,67 +28,84 @@ public void initialize() {
         Drivetrain.rightTalon1.enableCurrentLimit(false);
         Drivetrain.rightTalon2.enableCurrentLimit(false);
 
-        targetLocked = false;
+        Robot.hood.setHoodDown();
 
-        Globals.indexerEnabled = false;
+        targetLocked = false;
```

Comment:
```suggestion
        targetLocked = false;
```
```suggestion
        targetLocked = false;
        
        Globals.indexerEnabled = false;
```
In order for this feature to work in autos, this flag needs to be set